### DISCUSSION TITLE
Suggestion: Video support

### DISCUSSION THREAD

**Discussion by LexTypeC** [volotat/Anagnorisis#1] [Ideas]
Support for working with embeddings produced by [VideoCLIP](https://github.com/AskYoutubeAI/AskVideos-VideoCLIP) would be a potentially particularly interesting addition.

**Comment by volotat** [comment]
Totally agree. I do have it in the plans and some very basic functionality already working on the 'Videos' page. But before it I will probably refactor and rebuild the 'Music' page as it is already quite outdated. I also plan to publish the road-map of the project. Maybe not a strict version of it, just as I see it right now.

**Comment by volotat** [comment]
Since version [v0.4.8](https://github.com/volotat/Anagnorisis/commit/1e785cc3f31a4276670f3def0263fba88f3b39cb) the sematic video search completely integrated in the project and works exactly as expected, thanks to amazing [jina-embeddings-v5-omni](https://huggingface.co/jinaai/jina-embeddings-v5-omni-small) model.
